packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1269,6 +1269,10 @@ export class ContainerRuntime
 		return this;
```

Comment:
all these changes are just reverts in the test branch, so it will be back to what it is in main